Documentation/EyeTracking/EyeTracking_IsUserCalibrated.md

```diff
@@ -0,0 +1,91 @@
+![Screenshot from eye calibration notification](../Images/EyeTracking/mrtk_et_calibration_notification_example.jpg)
+
+## To be or not to be eye calibrated
+
+If eye tracking is a fundamental part of your app experience, you may wish to ensure that the user's eye calibration is valid.
+The main reason for it to be invalid is that the user chose to skip the eye tracking calibration. 
+This page describes:
+- Describes how to detect that a user is eye calibrated
+- Provides a sample for how to trigger a user notification to instruct the user to go through the eye calibration
+	- Automatically dismiss notification if eye calibration becomes valid
+	- Manually dismiss notification if user chooses to continue without calibration
+
+### How to detect the eye calibration state 
+
+The [MixedRealityToolkit.InputSystem.EyeGazeProvider](EyeTracking_EyeGazeProvider.md) provides a ```bool?``` property called ```IsEyeGazeValid```.
+It returns null if no information from the eye tracker is available yet.
```

Comment:
A quick note - I see a few places where we describe this (which is good!) but also this sets is up to get out of sync if we change this in the future - it would be easy to get one place fixed up but also not others.

This is a general problem when writing docs/guidance, and it's useful to have a single point of reference. This could either be solved by:

1) Referencing a specific definition/API page
2) Linking (xref-ing) to the the generated docs themselves which I believe also have this comment.

Anyways this one is less critical, maybe more of a thing to think about in the future (these are just things I've seen happen in the past, where docs start out great but if things get duplicated in multiple places it is super easy for us to miss updates in a specific location)

Reply:
Agreed. Can you point me to instructions on how to use the xref and API refs correctly? I'd like to be able to test it before submitting, but not sure how.

Reply:
I'll have to update the docs to show how to do this - let's follow up on this one afterward.